Jenkinsfile

```diff
@@ -25,6 +25,7 @@ pipeline {
         BUNDLE_IMG="${OPERATOR_BUNDLE_IMAGE_CANDIDATE_NAME}"
 
         AEROSPIKE_CUSTOM_INIT_REGISTRY="568976754000.dkr.ecr.ap-south-1.amazonaws.com"
+        AEROSPIKE_CUSTOM_INIT_REGISTRY_NAMESPACE="aerospike"
```

Comment:
Shouldn't we use a different namespace to test new changes?

Reply:
To use a different namespace, we have to push the image in ecr registry with a different namespace. Instead, I wrote a test to validate final image string some different namespace.